Tests/StoreKitUnitTests/ProductsManagerTests.swift

```diff
@@ -156,13 +156,13 @@ class SK1ProductsManagerDiagnosticsTrackingTests: ProductsManagerTests {
         }
 
         expect(self.mockDiagnosticsTracker.trackedProductsRequestParams.value).toEventually(haveCount(1))
-        let params = self.mockDiagnosticsTracker.trackedProductsRequestParams.value.first
-        expect(params?.wasSuccessful) == true
-        expect(params?.storeKitVersion) == .storeKit1
-        expect(params?.requestedProductIds) == Set([identifier, notFoundIdentifier])
-        expect(params?.notFoundProductIds) == Set([notFoundIdentifier])
-        expect(params?.errorMessage).to(beNil())
-        expect(params?.errorCode).to(beNil())
+        let params = try XCTUnwrap(self.mockDiagnosticsTracker.trackedProductsRequestParams.value.first)
+        expect(params.wasSuccessful) == true
+        expect(params.storeKitVersion) == .storeKit1
+        expect(Set(params.requestedProductIds)) == [identifier, notFoundIdentifier]
+        expect(params.notFoundProductIds) == [notFoundIdentifier]
```

Comment:
Should this also be a `Set`?
```suggestion
        expect(Set(params.notFoundProductIds)) == [notFoundIdentifier]
```

Reply:
Shouldn't really matter with one element I guess, but to properly compare it I think it's best to have it :)

Reply:
Exactly. But you're right that, for consistency, better to have it like that! Thanks!